test/lib/commands/outdated.js

```diff
@@ -343,7 +343,9 @@ t.test('should return if no outdated deps', async t => {
   })
 
   await outdated(testDir, {
-    global: false,
+    config: {
```

Comment:
these tests were passing entirely by coincidence. if they had been setting `global` to `true` they would not have done the expected thing